eng/precommit_local_build.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Python version 3.4 or higher is required to run this script.
+
+# Use case: Runs compilation, testing, and linting for the passed artifacts and POM artifacts.
+#
+# The artifacts must exist in the module list of pom.xml in the root of azure-sdk-for-java. Artifacts may be passed in two ways:
+#
+# 1. Comma separated list of relative POM paths.
+# 2. Comma separated list of groupId:artifactId identifiers.
+#
+# The script must be run at the root of azure-sdk-for-java.
+
+import argparse
+import os
+import xml.etree.ElementTree as ET
+
+baseCommand = 'mvn clean install -f pom.xml -pl "{}" -am {}'
```

Comment:
I think we should just disable `gpg` by default. I don't think we need to run gpg check locally.

Reply:
Good catch